libcontainer/init_linux.go

```diff
@@ -149,7 +161,7 @@ func StartInitialization() (retErr error) {
 	}()
 
 	// If init succeeds, it will not return, hence none of the defers will be called.
-	return containerInit(it, pipe, consoleSocket, fifofd, logPipeFd, mountFds)
+	return containerInit(it, pipe, consoleSocket, fifofd, logFD, mountFds)
```

Comment:
OK, we now use 3 different ways to say "FD", but I'm pretty sure the last one is the best one.

Reply:
I personally prefer Fd. I can write up a PR to unify this.